crates/ruff/src/rules/wemake_python_styleguide/helpers/naming.rs

```diff
@@ -0,0 +1,78 @@
+use std::path::Path;
+
+use once_cell::sync::Lazy;
+use regex::Regex;
+
+const ALIAS_NAMES_WHITELIST: [&str; 7] = ["np", "pd", "df", "plt", "sns", "tf", "cv"];
```

Comment:
```suggestion
const ALIAS_NAMES_ALLOWLIST: [&str; 7] = ["np", "pd", "df", "plt", "sns", "tf", "cv"];
```

Or what I like to do (if you don't need to enumerate the values) is to write a `match` statement instead:

```
if matches!(x, "np" | "pd" | "df" | ...)
```

The compiler will generate efficient code for this

Reply:
I wasn't sure what to do here, because it is a potential conflict with another linter. flake8-import-conventions defines convential import aliases [here](https://github.com/astral-sh/ruff/blob/e574a6a7698a83d68bd42b27b1ba7a304e09f808/crates/ruff/src/rules/flake8_import_conventions/settings.rs#L8). Actually I would like to use these as the single source of truth for allowed aliases. This list is also configurable via settings (see [here](https://github.com/astral-sh/ruff/blob/e574a6a7698a83d68bd42b27b1ba7a304e09f808/crates/ruff/src/rules/flake8_import_conventions/settings.rs#L50-L59)) in which case I would also like to add these to the allowlist.

How should this be done?

Reply:
@charliermarsh ?